py/desiutil/test/test_install.py

```diff
@@ -326,7 +326,7 @@ def test_set_install_dir(self):
         with patch.dict('os.environ', {'NERSC_HOST': 'edison', 'DESI_PRODUCT_ROOT': 'FAKE'}):
             del environ['DESI_PRODUCT_ROOT']
             test_code_version = 'test-blat-foo'
-            options = self.desiInstall.get_options(['desiutil', test_code_version])
+            options = self.desiInstall.get_options(['desiutil', test_code_version, '--test'])
```

Comment:
Could you explain in more detail why this change was necessary?

Reply:
`set_install_dir()` will now raise an exception if the installation directory doesn't exist, which was tripping up the test that was using the fallback to the NERSC default location, which doesn't exist in the test environment.  When actually installing at NERSC, I think it should be an error if the default location doesn't exist, so I used the `--test` hooks to prevent the existence check exception in `set_install_dir` while testing.

Reply:
OK.  I'll take a look and see if the unit test can be made a bit more robust.